kong/plugins/jwt/handler.lua

```diff
@@ -127,7 +127,7 @@ local function do_authentication(conf)
   local token, err = retrieve_token(conf)
   if err then
     kong.log.err(err)
-    return kong.response.exit(500, { message = "An unexpected error occurred" })
+    return kong.response.error(500, "An unexpected error occurred")
```

Comment:
In cases where plugins use the default error message, they should not specify it.

```suggestion
    return kong.response.error(500)
```

OR even:
```suggestion
    return kong.response.error(ngx.HTTP_INTERNAL_SERVER_ERROR)
```